frontend/src/components/TransactionsPerSecond/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo } from "react";
+import { useDispatch } from "react-redux";
+import BigNumber from "bignumber.js";
+
+import { useRedux } from "hooks/useRedux";
+import {
+  ActionStatus,
+  LedgerTransactionHistoryFilterType,
+  Network,
+} from "types";
+import { SectionCard } from "components/SectionCard";
+import {
+  VerticalBarChart,
+  TimeRange,
+} from "components/charts/VerticalBarChart";
+import { fetchTransactionsHistoryAction } from "ducks/transactions";
+import "./styles.scss";
+import { AmountInfoCard } from "components/AmountInfoCard";
+import { Icon } from "@stellar/design-system";
+
+export const TransactionsPerSecond = () => {
+  const { transactions, ledgers } = useRedux("transactions", "ledgers");
+  const dispatch = useDispatch();
+  const lastLedger = ledgers.lastLedgerRecords[0];
+  const selectedTimeInterval = LedgerTransactionHistoryFilterType["30D"];
+
+  useEffect(() => {
+    dispatch(fetchTransactionsHistoryAction(Network.MAINNET));
+  }, [dispatch, selectedTimeInterval]);
+
+  const data = useMemo(() => {
+    const result = transactions.transactionsHistory.items.map((item) => ({
+      date: new Date(item.date),
+      primaryValue: new BigNumber(
+        new BigNumber(item.txTransactionCount / 84600).toFormat(2),
```

Comment:
this is also dividing the current day by `84600` which will skew results, since the current full day hasn't passed (unless you load this page right at the end of the day)

how about instead we use the `end` data that's returned from the backend to get the total time that's passed?

We'll have to also add the `end` data that's returned from the backend in the duck/transactions.ts